### PR TITLE
Ability to export repositories in V1 format

### PR DESCRIPTION
**Pull request by adutra**

### CODE REVIEW

**Review by dimas-b — COMMENTED** [review]
LGTM overall :+1: I think it's a useful feature.

Could you maybe add a small `IT*` for this, just to smoke test in case our core changes break this in the new model in the future?

File: servers/quarkus-cli/src/main/java/org/projectnessie/quarkus/cli/ExportRepository.java [hunk 1]
```diff
@@ -127,6 +128,14 @@ enum Format {
               + ".")
   private Integer commitBatchSize;
 
+  @CommandLine.Option(
+      names = EXPORT_VERSION,
+      description =
+          "The export version, defaults to "
+              + ExportImportConstants.DEFAULT_EXPORT_VERSION
```
[the remark this answers is not in the document]
Comment (adutra) [reply servers/quarkus-cli/src/main/java/org/projectnessie/quarkus/cli/ExportRepository.java]: I don't think so, there are other constant in the same situation – but you made me realize that I should be using the constant in a few other places, will update the PR.

[the remark this answers is not in the document]
Comment (dimas-b) [reply servers/quarkus-cli/src/main/java/org/projectnessie/quarkus/cli/ExportRepository.java]: fair enough

**Review by dimas-b — APPROVED** [review]

File: servers/quarkus-cli/src/main/java/org/projectnessie/quarkus/cli/ExportRepository.java [hunk 2]
```diff
@@ -130,6 +130,7 @@ enum Format {
 
   @CommandLine.Option(
       names = EXPORT_VERSION,
+      defaultValue = "2",
```
Comment (dimas-b) [inline servers/quarkus-cli/src/main/java/org/projectnessie/quarkus/cli/ExportRepository.java:133]: nit: `ExportImportConstants.DEFAULT_EXPORT_VERSION` ?

**Review by dimas-b — DISMISSED** [review]

File: servers/quarkus-cli/src/main/java/org/projectnessie/quarkus/cli/ExportRepository.java (hunk 2 above)
Comment (adutra) [reply servers/quarkus-cli/src/main/java/org/projectnessie/quarkus/cli/ExportRepository.java]: You are right 🤦

**Review by dimas-b — APPROVED** [review]

**Pull request merged**